lib/slimmer/app.rb

```diff
@@ -110,5 +110,9 @@ def rewrite_response(env, response)
     def strip_slimmer_headers(headers)
       headers.reject {|k, v| k =~ /\A#{Headers::HEADER_PREFIX}/ }
     end
+
+    def application_class_name
+      @app.class.parent_name if @app.class.respond_to?(:parent_name)
```

Comment:
It's a little vague to me what sort of return value we'd expect here, i assume something like `FinderFrontend`?